src/services/layerHelper.js

```diff
@@ -48,7 +48,7 @@ ngeo.LayerHelper.GROUP_KEY = 'groupName';
  * @param {string} sourceURL The source URL.
  * @param {string} sourceLayersName A dot separated names string.
  * @param {string=} opt_serverType Type of the server ("mapserver",
- *     "geoserver", qgisserver, …).
+ *     "geoserver", "qgis", …).
```

Comment:
For me ol3 is wrong :-( 
see: http://docs.qgis.org/2.8/fr/docs/user_manual/preamble/whats_new.html#qgis-server